toolbox

```diff
@@ -1040,9 +1041,11 @@ list_images()
         return 1
     fi
 
-    if ! output=$(echo "$details" \
-                  | sed "s/ \{2,\}/\t/g" 2>&3 \
-                  | column --separator "$tab" --table --table-columns "IMAGE ID,IMAGE NAME,CREATED" 2>&3); then
+    table_data=$(printf "%s\t%s\t%s\n" "IMAGE ID" "IMAGE NAME" "CREATED"; \
+                 echo "$details" \
+                 | sed "s/ \{2,\}/\t/g" 2>&3)
```

Comment:
Nitpick: it would be nice if we could also handle errors from the `sed` invocation. I am just paranoid because of the inherent fragility of parsing output from random commands (eg., `details`). One easy way to do that would be to move it into the block below.